adventure/less/__init__.py

```diff
@@ -161,12 +160,35 @@ def forced_move():
         check.stdout("> ")
         check.stdin(move, prompt=False)
 
-    check.stdout("You find yourself at the edge of a impassible stream. You are forced to return where you came from.",
+    check.stdout("You find yourself at the edge of a impassible stream. You head back to the depression.",
```

Comment:
Waarom moest dit?

Reply:
In de nieuwe datafiles staat deze beschrijving, die blijkbaar afwijkt van wat er eerst stond.

Reply:
Hmm dat is wel een beetje gek. Ik heb ze vers gedownload van de eigen site en toen getransformeerd naar het nieuwe formaat :-) Nu ja, laten we er van uit gaan dat het goed is zo.

Reply:
Dan is het waarschijnlijk eerder een keer veranderd, want die omschrijving die er eerst stond is niet te vinden in de datafiles, haha.